cli/go.mod

```diff
@@ -77,6 +77,7 @@ require (
 	github.com/muesli/termenv v0.15.2 // indirect
 	github.com/oklog/ulid v1.3.1 // indirect
 	github.com/pelletier/go-toml v1.9.3 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
Please remove this package as its not needed anymore in golang. 

Go error management already has this

Reply:
Thank you for catching that! I completely missed that this functionality had been added to the standard library. I just updated the PR and removed the dependency